### DISCUSSION TITLE
Portrait oriented monitors and Sixths

### DISCUSSION THREAD

**Discussion by affrae** [rxhanson/RectanglePro-Community#109] [General]
Hi I seem to get a little disoriented with how Rectangle Pro handles sixths when a window is set to portrait. For my landscape monitor I get the following for sixths when I assign a modifier and keys 1-6:

- Top left
- Top Middle
- Top Right
- Bottom Left
- Bottom Middle
- Bottom Right

On my portrait monitor I get the following for sixths with the same modifier and keys 1-6:

- Top left
- Middle Left
- Top Right
- Bottom Left
- Middle Right
- Bottom Right

I think I get what is going open there, but it feels wrong - is it possible for Rectangle Pro to run it as follows:

- Top left
- Top Right
- Middle Left
- Middle Right
- Bottom Left
- Bottom Right

Or is this related to #27

??

**Comment by rxhanson** [comment]
I think I see what you mean here. Maybe at a start I'll add in a terminal command to adjust this since I don't think you're the first to get tripped up by it.
